hack/release-scripts/generate-release-pr

```diff
@@ -88,10 +88,14 @@ parse_args() {
 
 update_readme() {
   log "Updating README.md"
-  # vi macro that adds new driver version 'Container Images' row to README.md
-  vi -s <(echo "gg/## Container Images
-  jjjjyy:%s/${PREV_DRIVER_VERSION}/${NEW_DRIVER_VERSION}/g
-  jjjjjjp:wq") "$README_PATH"
+  $SED "/^## Container Images$/,+5 s/$PREV_DRIVER_VERSION/$NEW_DRIVER_VERSION/" -i "$README_PATH"
+
+  # If the major/minor version has changed, we also need to replace the old prev version with the new previous version
+  # (If not, that means this is a patch release, and we shouldn't change the other version)
+  if [ "${PREV_DRIVER_VERSION%.*}" != "${NEW_DRIVER_VERSION%.*}" ]; then
+    OLD_VERSION_TO_REPLACE=$($SED -n "/^## Container Images$/,+5 p" "$README_PATH" | tail -n 1 | grep -o " v[^ ]* " | tr -d " ")
+    $SED "/^## Container Images$/,+5 s/$OLD_VERSION_TO_REPLACE/$PREV_DRIVER_VERSION/" -i "$README_PATH"
```

Comment:
You only update the Driver Version column, not the image tags themselves. 

`> ./hack/release-scripts/generate-release-pr v1.28.0 v1.29.0`

```diff
-| v1.28.0        | registry.k8s.io/provider-aws/aws-ebs-csi-driver:v1.28.0                                           | public.ecr.aws/ebs-csi-driver/aws-ebs-c
si-driver:v1.28.0                      |
-| v1.27.0        | registry.k8s.io/provider-aws/aws-ebs-csi-driver:v1.27.0                                           | public.ecr.aws/ebs-csi-driver/aws-ebs-c
si-driver:v1.27.0                      |
+| v1.29.0        | registry.k8s.io/provider-aws/aws-ebs-csi-driver:v1.28.0                                           | public.ecr.aws/ebs-csi-driver/aws-ebs-c
si-driver:v1.28.0                      |
+| v1.28.0        | registry.k8s.io/provider-aws/aws-ebs-csi-driver:v1.27.0                                           | public.ecr.aws/ebs-csi-driver/aws-ebs-c
si-driver:v1.27.0
```

Reply:
Should be fixed in latest revision